reference-app/src/main/java/org/smartregister/anc/interactor/LoginInteractor.java

```diff
@@ -33,6 +34,9 @@ protected void scheduleJobsPeriodically() {
         SyncSettingsServiceJob
                 .scheduleJob(SyncSettingsServiceJob.TAG, TimeUnit.MINUTES.toMillis(BuildConfig.CLIENT_SETTINGS_SYNC_MINUTES),
                         getFlexValue(BuildConfig.CLIENT_SETTINGS_SYNC_MINUTES));
+        DocumentConfigurationServiceJob
+                .scheduleJob(DocumentConfigurationServiceJob.TAG, TimeUnit.MINUTES.toMillis(BuildConfig.CLIENT_SETTINGS_SYNC_MINUTES),
```

Comment:
Can you add the job to run immediately after login?? I'll see if I can direct you to the exact location

Reply:
@ekigamba In the `LoginInteractor#scheduleJobsImmediately()` ? :thinking:

Reply:
@allan-allay Yea, I think that's it. Also add it to run when the sync button on the register is clicked

Reply:
Cool. Let me implement the changes